src/vlc.h

```diff
@@ -17,8 +17,8 @@ You should have received a copy of the GNU General Public License
 along with vlc-bittorrent.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#ifndef VLC_BITTORRENT_VLC_H
-#define VLC_BITTORRENT_VLC_H
```

Comment:
Using a macro starting with two underscores followed by capital letter conflict with the reserved name space of C and C++.  Perhaps best to avoid?